<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadianceReferenceDialog</class>
 <widget class="QDialog" name="RadianceReferenceDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reference</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="nuclidesTabWidget">
      <attribute name="title">
       <string>Nuclides</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="nuclideTopLayout">
         <item>
          <widget class="QComboBox" name="nuclideComboBox">
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
           <property name="minimumContentsLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editNuclideButton">
           <property name="text">
            <string>Edit...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/radiance.qrc">
             <normaloff>:/Radiance/icons/page_white_edit.png</normaloff>:/Radiance/icons/page_white_edit.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="nuclideTopLayoutSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addNuclideButton">
           <property name="text">
            <string>Add...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/radiance.qrc">
             <normaloff>:/Radiance/icons/page_white_add.png</normaloff>:/Radiance/icons/page_white_add.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolBox" name="nuclideCategoriesToolBox">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="nuclideGammaSpectrumPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>596</width>
            <height>326</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Gamma spectrum</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTableView" name="nuclideGammaSpectrumTableView">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="cornerButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="materialsTabWidget">
      <attribute name="title">
       <string>Materials</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="materialTopLayout">
         <item>
          <widget class="QComboBox" name="materialComboBox">
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
           <property name="minimumContentsLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editMaterialButton">
           <property name="text">
            <string>Edit...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/radiance.qrc">
             <normaloff>:/Radiance/icons/page_white_edit.png</normaloff>:/Radiance/icons/page_white_edit.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="materialTopLayoutSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addMaterialButton">
           <property name="text">
            <string>Add...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/radiance.qrc">
             <normaloff>:/Radiance/icons/page_white_add.png</normaloff>:/Radiance/icons/page_white_add.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
         <item>
          <widget class="QLabel" name="rhoLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>ρ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rhoValueLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolBox" name="materialCategoriesToolBox">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="materialDoseBuildupFactorPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>596</width>
            <height>278</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Dose buildup factor</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="top_label">
             <property name="text">
              <string>μd</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="doseBuildUpFactorLayout">
             <item>
              <widget class="QLabel" name="left_label">
               <property name="text">
                <string>E, MeV</string>
               </property>
               <property name="textFormat">
                <enum>Qt::PlainText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="materialDoseBuildupFactorTableView">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="tabKeyNavigation">
                <bool>false</bool>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="materialMassAttenuationCoefficientPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>89</width>
            <height>89</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Mass attenuation coefficient</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTableView" name="materialMassAttenuationCoefficientTableView">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="cornerButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Multi-layered materials</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="materialTopLayout_2" stretch="1,0,0">
         <item>
          <widget class="QComboBox" name="multiLayeredMaterialComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
           <property name="minimumContentsLength">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editMultiLayeredMaterialButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Edit...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/radiance.qrc">
             <normaloff>:/Radiance/icons/page_white_edit.png</normaloff>:/Radiance/icons/page_white_edit.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addMultiLayeredMaterialButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/radiance.qrc">
             <normaloff>:/Radiance/icons/page_white_add.png</normaloff>:/Radiance/icons/page_white_add.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolBox" name="nuclideCategoriesToolBox_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="multiLayeredMaterialLayersPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>596</width>
            <height>326</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Layers</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableView" name="materialLayersTableView">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="cornerButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/radiance.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>RadianceReferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
